CMakeLists.txt

```diff
@@ -311,7 +314,7 @@ endif()
 # FIXME: Support ARM "-mfpu=vfp -mfloat-abi=hard"
 if (("${TARGET_ARCHITECTURE}" STREQUAL "x86_64") OR ("${TARGET_ARCHITECTURE}" STREQUAL "i686"))
   if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
-    set(SSE_FLAGS "-mfpmath=sse" "-msse" "-msse2")
+    set(SSE_FLAGS "-mfpmath=sse" "-msse" "-msse2" "-std=c++0x")
```

Comment:
Okay so something like this in the appropriate place in the top-level `CMakeLists.txt` file would do the job.

```cmake
################################################################################
# C++ language version
################################################################################
# FIXME: Use CMake's own mechanism for selecting language version
if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
  z3_add_cxx_flag("-std=c++11" REQUIRED)
else()
  message(AUTHOR_WARNING "Not setting C++ language version for compiler")
endif()
```